pyanaconda/exception.py

```diff
@@ -248,7 +248,8 @@ def initExceptionHandling(anaconda):
                  "/tmp/program.log", "/tmp/storage.log", "/tmp/ifcfg.log",
                  "/tmp/dnf.librepo.log", "/tmp/hawkey.log",
                  "/tmp/lvm.log", iutil.getSysroot() + "/root/install.log",
-                 "/proc/cmdline", "/root/lorax-packages.log"]
+                 "/proc/cmdline", "/root/lorax-packages.log",
+                 "/tmp/blivet-gui-utils.log"]
```

Comment:
I plan to add a separate log for the GUI process later. This one is just for the daemon ("BlivetUtils") process.